packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -76,6 +76,17 @@ type Props = {|
      */
     required?: boolean,
 
+    /**
+     * The error message that should render if this field is
+     * required and left blank. Note that this message will not be used
+     * if a `validate` prop is passed in. Also note that error messages
+     * only redner for the LabeledTextField component, not for TextField.
+     *
+     * Implementation note: Added this field so that users can pass in
+     * translated strings.
+     */
+    requiredErrorMessage: string,
```

Comment:
See my earlier comment regarding the `required` field and this one being combined.